examples/src/main/scala/Simple.scala

```diff
@@ -79,5 +80,6 @@ object SimpleMain extends App {
   finally {
     transport.shutdown()
     server()
+    pool.shutdown()
```

Comment:
Hmm seeing this, it looks like we should not be making users clean up after us. Ideally we should be cleaning up after ourselves - i.e. internally close the thread pool when we're shutting down. WDYT?